docs/core/entity/sensor.md

```diff
@@ -48,6 +48,7 @@ If specifying a device class, your sensor entity will need to also return the co
 | energy | Wh,kWh | Energy.
 | power_factor | % | Power Factor.
 | voltage | V | Voltage.
+| volatile_organic_compounds | µg/m³ | Concentration of volatile organic compounds.
```

Comment:
If you have time, it would be great to have this table sorted 🔡.

Reply:
Done in https://github.com/home-assistant/developers.home-assistant/pull/1036/commits/d73fbc8073b70bee595ef608dc24db90676f7590